tensorflow/lite/micro/micro_allocator_test.cc

```diff
@@ -115,7 +116,7 @@ void AllocateAndVerifyMockWeightTensor(
 void EnsureUniqueVariableTensorBuffer(const Model* model,
                                       TfLiteEvalTensor* eval_tensors,
                                       const int variable_tensor_idx) {
-  for (size_t i = 0; i < GetModelTensorCount(model); i++) {
+  for (size_t i = 0; i < GetModelTensorCount(model, 0); i++) {
```

Comment:
nit: /*subgraph_idx=*/0?

Reply:
and other place in which GetModelTensorCount is called. On another thought, if you have to change every place with a new argument, might just change the function to GetSubgraphTensorCountFromModel(). Up to you though